ironfish/src/primitives/target.test.ts

```diff
@@ -121,11 +121,29 @@ describe('Calculate target', () => {
 
       expect(newTarget.targetValue).toBeLessThan(target.targetValue)
     }
+
+    // for any time 35-45 seconds after the last block, difficulty should increase by previous block's difficulty / BigInt(2048) * 2
```

Comment:
why is 35-45 seconds special?

Reply:
Previously when the target was 15 -> then the difficulty had increased from 1 to 10
but right now the difficulty will be increased by bucket length pattern
...
15 - 25 - * 2048 * 4
25 - 35 - * 2048 * 3
35 - 45 - * 2048 * 2
45 - 55 - * 2048
55 - 64 - nothing
...

Reply:
ok great, we should have tests to reflect that then.